benchmarks/util/result_parser.py

```diff
@@ -252,5 +284,11 @@ def get_result_from_file(cpp_file="",
     __parse_go_result(go_file)
   if synthetic_file != "":
     __parse_synthetic_result(synthetic_file)
+  if node_file != "":
+    __parse_js_php_result(node_file, "node")
```

Comment:
Why php and js share the same method?

Reply:
Their results have the same json format (which are constructed by myself). For code reuse I just use the same method to parse the result.

Reply:
Could you give it a better name? e.g., __parse_XX_format_result and let js and php wrap it?
Why other parse methods work with different json format?

Reply:
Done. The other results are most like using third_party benchmark tools for benchmark, e.g. c++ is using google/benchmark, the json result format are defined by the tool. But js and php and just using language internal time library to get the consuming time.